data/scopes/gigaset-Maxwell2.ini

```diff
@@ -7,9 +7,12 @@ provisioning_url_filename = "%25MACD.xml"
 tmpl_phone = "gigaset-Maxwell.tmpl"
 cap_linekey_count = 8
 cap_linekey_type_blacklist =
-cap_softkey_count =
+cap_softkey_count = 2
 cap_softkey_type_blacklist =
```

Comment:
Add the list of softkey types that are not supported. From your proposed macro I understand only "redial" and "pickup" are allowed. All the others must be added to the blacklist ` cap_softkey_type_blacklist =`